core/utils/cuckoo_map_test.cc

```diff
@@ -45,6 +45,35 @@ TEST(CuckooMapTest, Insert) {
   EXPECT_EQ(cuckoo.Insert(1, 1)->second, 1);
 }
 
+struct Foo {
+  Foo() : a(), b(), c() {}
+
```

Comment:
Could you delete the default copy constructor and copy assignment operator? For the purpose of the test, it would be better for Foo to avoid accidental copy.

Reply:
Ah, sure-- this will require a few non-trivial changes to CuckooMap, though. It currently assumes value types are copyable and assignable (and now moveable)